src/ansi-c/library/x86_assembler.c

```diff
@@ -3,30 +3,30 @@
 
 extern int __CPROVER_rounding_mode;
 
-void __asm_fnstcw(unsigned short *dest)
+void __asm_fnstcw(void *dest)
```

Comment:
Wozu denn diese Änderung?

Reply:
`remove_asmt::gcc_asm_function_call` produces function-type symbols with `void *` parameter types, and the above change avoids a linker warning. (Test execution on FreeBSD actually triggers this code path via `fesetround`.)

Reply:
Change removed, will PR separately.